benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -120,6 +125,24 @@ void GraphicsBenchmarkApp::Setup()
         mGraphicsPipelinesIndexer.AddDimension(kAvailableVbFormats.size());
         mGraphicsPipelinesIndexer.AddDimension(kAvailableVertexAttrLayouts.size());
     }
+    // Samplers
+    {
+        grfx::SamplerCreateInfo samplerCreateInfo = {};
+        samplerCreateInfo.magFilter               = grfx::FILTER_LINEAR;
+        samplerCreateInfo.minFilter               = grfx::FILTER_LINEAR;
+        samplerCreateInfo.mipmapMode              = grfx::SAMPLER_MIPMAP_MODE_LINEAR;
+        samplerCreateInfo.minLod                  = 0;
+        samplerCreateInfo.maxLod                  = FLT_MAX;
+        PPX_CHECKED_CALL(GetDevice()->CreateSampler(&samplerCreateInfo, &mLinearSampler));
+
+        samplerCreateInfo            = {};
+        samplerCreateInfo.magFilter  = grfx::FILTER_NEAREST;
+        samplerCreateInfo.minFilter  = grfx::FILTER_NEAREST;
+        samplerCreateInfo.mipmapMode = grfx::SAMPLER_MIPMAP_MODE_NEAREST;
+        samplerCreateInfo.minLod     = 0.0f;
+        samplerCreateInfo.maxLod     = 1.0f;
```

Comment:
I am curious, is there a reason maxLod is 1 here and FLT_MAX for linear?

Reply:
in the case of 1x1 texture, this value doesn't really matter since there is only 1 mip level. But actually we could just use the linear sampler directly for the 1x1 case. No need to switch the sampler in the 1x1 case

Reply:
I see, OK. I've removed `mNearestSampler`